frontend/src/modules/Dashboard/Styles/GroupCard.style.tsx

```diff
@@ -44,3 +57,6 @@ export const StyledButtons = styled.div`
   display: flex;
   justify-content: space-around;
 `
+interface ColorProps {
+  color: string
+}
```

Comment:
I think here can be moved to one of the "Types" files, likely for "frontend/src/modules/Core/Types/StyledComponent.ts"